zuul-core/src/main/java/com/netflix/zuul/netty/server/http2/Http2SslChannelInitializer.java

```diff
@@ -51,18 +51,6 @@
     private final SwallowSomeHttp2ExceptionsHandler swallowSomeHttp2ExceptionsHandler;
     private final String metricId;
 
-
-    /**
-     * Use {@link #Http2SslChannelInitializer(String, ChannelConfig, ChannelConfig, ChannelGroup)} instead.
```

Comment:
The reason this is deprecated (and not deleted) is because this is a backward incompatible change.

Reply:
Well, the reason we deprecate is to eventually be able to remove. 
I see this was only deprecated recently. We should probably make the deprecation window explicit.

Reply:
Yeah, I would cut a minor release, then probably make this change.  It's not actually semver, but it's enough to be able to upgrade.